src/components/ClosestVenues.js

```diff
@@ -57,8 +58,8 @@ class ClosestVenues extends Component {
                 return (
                   <li key={i}>
```

Comment:
Using map index on components that are re-used ends up with weird behaviour in react. use the venue.id for the key instead to avoid.